Formula/l/lima-additional-guestagents.rb

```diff
@@ -0,0 +1,38 @@
+class LimaAdditionalGuestagents < Formula
+  desc "Additional guest agents for Lima"
+  homepage "https://lima-vm.io/"
+  url "https://github.com/lima-vm/lima/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "0c5e601c3788eaaa9f45bdf00d69e23b5720dd2f35e1be36ad21c032fdabf738"
+  license "Apache-2.0"
+  head "https://github.com/lima-vm/lima.git", branch: "master"
+
+  depends_on "go" => :build
+  depends_on "lima"
+
+  # PR https://github.com/lima-vm/lima/pull/3564
+  # For https://github.com/Homebrew/homebrew-core/pull/224206#issuecomment-2896811998
+  # > Minitest::Assertion: Expected
+  # > {"aarch64" => {"location" => "/home/linuxbrew/.linuxbrew/Cellar/lima/1.1.0/share/lima/
+  # > lima-guestagent.Linux-aarch64.gz"}} to include "riscv64".}
+  patch do
+    url "https://github.com/lima-vm/lima/commit/230e36c359aa999c24f8a8c15045877342b2f231.patch?full_index=1"
+    sha256 "e4728c62d228b5f3c678ff85c723af0918b65dc8d1d777f1ea29fe619b0ff9b3"
+  end
+
+  def install
+    if build.head?
+      system "make", "additional-guestagents"
+    else
+      # VERSION has to be explicitly specified when building from tar.gz, as it does not contain git tags
+      system "make", "additional-guestagents", "VERSION=#{version}"
+    end
+
+    bin.install Dir["_output/bin/*"]
+    share.install Dir["_output/share/*"]
+  end
+
+  test do
+    info = JSON.parse shell_output("#{HOMEBREW_PREFIX}/bin/limactl info")
```

Comment:
should be fine without the prefix setup

```suggestion
    info = JSON.parse shell_output("limactl info")
```

Reply:
done